<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionReservation</class>
 <widget class="QMainWindow" name="GestionReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background:rgb(46,0,127);</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1071</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid#9708B4;
    border-bottom-color: #9708B4; /* same as the pane color */
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    min-width:60ex;
    padding: 2px;
     font: bold, &quot;Comic Sans MS&quot;;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #085394, stop: 0.4 #9708B4,
                                stop: 0.5 #9708B4, stop: 1.0 #9708B4);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #9708B4; /* same as pane color */
}
QLabel{
background:transparent;
font:Century gothic;
font-size:20px;
color:#8b8b8b;
}

QDateEdit{
background:#8b8b8b;
}
QGroupBox{
font-color:rgb(255, 255, 255);
}



</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ajouter_reservation">
     <attribute name="title">
      <string>Ajouter une reservation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>611</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de reservation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de reservation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de salle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>190</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID de client</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_reservation">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="Type_reservation">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>VIP/Normal</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_reservation">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="Date_reservation">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>230</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_combo">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>150</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border: 3px solid white;
}
QPushButton:hover {
color:#fff;
border:15px;
background: rgb(198, 156, 109);
}</string>
       </property>
       <property name="text">
        <string>SALLES</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_combo_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>190</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border: 3px solid white;
}
QPushButton:hover {
color:#fff;
border:15px;
background: rgb(198, 156, 109);
}</string>
       </property>
       <property name="text">
        <string>CLIENT</string>
       </property>
      </widget>
      <widget class="QLabel" name="labela">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>240</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Status de carte TAG</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimer_reservation">
     <attribute name="title">
      <string>Supprimer une reservation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_reservation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID_reservation_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>130</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Afficher_reservation">
     <attribute name="title">
      <string>Afficher une reservation</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recherche et Tri</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_chercher">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ID_RESERVATION</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_CLIENT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID_SALLE</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rech">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tri">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>TYPE_RESERVATION</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>591</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>PDF</string>
     </attribute>
     <widget class="QTextEdit" name="text_pdf">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
